network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -263,16 +264,16 @@ <h2 class="h3-heading">{% trans "Company contact info" %}</h3>
     <div class="container">
       <div class="row">
         <div class="{{section_class}} mt-5 pt-3 bg-white product-section-padding-x">
-          {% if product.updates|length > 0 or coralTalkServerUrl %}
+          {% if product.product_updates.count > 0 or coralTalkServerUrl %}
           <h3 class="h2-heading mb-4 mb-md-5">{% trans "Updates" %}</h3>
           {% endif %}
-          {% if product.updates|length > 0 %}
+          {% if product.product_updates.count > 0 %}
           <div class="mb-5">
-            {% for product in product.updates %}
+            {% for product in product.product_updates.all %}
               <div class="product-update mb-4">
-                <a class="product-update-link h5-heading title" href="{{product.source}}" target="_blank">{{product.title}}</a>
-                <div class="author">{{product.author}}</div>
-                <div class="snippet">{{product.snippet}}</div>
+                <a class="product-update-link h5-heading title" href="{{product.update.source}}" target="_blank">{{product.update.title}}</a>
+                <div class="author">{{product.update.author}}</div>
+                <div class="snippet">{{product.update.snippet}}</div>
```

Comment:
this doesn't feel great: updates are not products, so we should definitely not be saying `for product in ...` here. Ideally this is just `for update in updates`, but I don't think we can "unpack" the content of the many to many relation that way, so at the very least let's turn this into `for item in  updates` and then either accessing `item.update` everywhere, or using a `with update=item.update` like how you've done `related_product`.